sklearn/kernel_approximation.py

```diff
@@ -245,9 +245,8 @@ class AdditiveChi2Sampler(BaseEstimator, TransformerMixin):
     References
     ----------
     See `"Efficient additive kernels via explicit feature maps"
-    <http://eprints.pascal-network.org/archive/00006964/01/vedaldi10.pdf>`_
+    <http://citeseer.ist.psu.edu/viewdoc/summary?doi=10.1.1.167.7024>`_
```

Comment:
If you're going to change to the journal, please do so this PR can be disappeared!